<ui version="4.0" >
 <class>mainSettingsClass</class>
 <widget class="QWidget" name="mainSettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>mainSettings</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabMainSettings" >
     <property name="tabPosition" >
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <property name="elideMode" >
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="mainTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>424</width>
        <height>380</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Main</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../qutim.qrc" >
        <normaloff>:/icons/crystal_project/settings.png</normaloff>:/icons/crystal_project/settings.png</iconset>
      </attribute>
      <attribute name="toolTip" >
       <string>Main Window Settings</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QCheckBox" name="sizePosBox" >
         <property name="text" >
          <string>Save main window size and position</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="hideBox" >
         <property name="text" >
          <string>Hide on startup</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QCheckBox" name="loginDialogBox" >
         <property name="text" >
          <string>Show login dialog on startup</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QCheckBox" name="menuAccountBox" >
         <property name="text" >
          <string>Add accounts to system tray menu</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QCheckBox" name="showSettingsBox" >
           <property name="text" >
            <string>Show settings by default from:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="accountBox" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" >
        <widget class="QCheckBox" name="onTopBox" >
         <property name="text" >
          <string>Always on top</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QCheckBox" name="ahideBox" >
           <property name="text" >
            <string>Auto-hide:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="secondsBox" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="layoutDirection" >
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix" >
            <string> s</string>
           </property>
           <property name="maximum" >
            <number>3600</number>
           </property>
           <property name="value" >
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="8" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>360</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" >
        <widget class="QCheckBox" name="rowBox" >
         <property name="text" >
          <string>Draw the background using alternating colors</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cListTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>424</width>
        <height>380</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Contact List</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../qutim.qrc" >
        <normaloff>:/icons/crystal_project/contactlist.png</normaloff>:/icons/crystal_project/contactlist.png</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <item row="1" column="0" >
        <widget class="QGroupBox" name="clAccountsBox" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="title" >
          <string>Merge active accounts</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>6</number>
          </property>
          <property name="spacing" >
           <number>3</number>
          </property>
          <item row="0" column="0" >
           <layout class="QGridLayout" name="gridLayout" >
            <property name="spacing" >
             <number>3</number>
            </property>
            <item row="0" column="0" >
             <widget class="QCheckBox" name="showGroupsBox" >
              <property name="text" >
               <string>Show groups</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="hideGroupsBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Hide empty groups</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QCheckBox" name="offlineBox" >
              <property name="text" >
               <string>Hide offline users</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QCheckBox" name="separatorBox" >
              <property name="text" >
               <string>Hide online/offline separators</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="sortBox" >
         <property name="title" >
          <string>Sort by:</string>
         </property>
         <layout class="QGridLayout" >
          <property name="verticalSpacing" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QRadioButton" name="unsortedButton" >
            <property name="text" >
             <string>Unsorted</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QRadioButton" name="byStatusButton" >
            <property name="text" >
             <string>Sort by status</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QRadioButton" name="byNameButton" >
            <property name="text" >
             <string>Sort by name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QCheckBox" name="clientColumnBox" >
         <property name="text" >
          <string>Show client icons</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="clLnFGrouBox" >
         <property name="title" >
          <string>Look'n'Feel</string>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <property name="margin" >
           <number>5</number>
          </property>
          <property name="spacing" >
           <number>2</number>
          </property>
          <item row="0" column="0" >
           <layout class="QHBoxLayout" name="clLnFLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="clOpacityLbl" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Opacity:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSlider" name="clOpacitySlider" >
              <property name="maximum" >
               <number>100</number>
              </property>
              <property name="value" >
               <number>100</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition" >
               <enum>QSlider::TicksAbove</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="clOpacityPersLbl" >
              <property name="minimumSize" >
               <size>
                <width>35</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>35</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>100%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLabel" name="clWinStyleLbl" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Window Style:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="clWinStyleComboBox" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text" >
                <string>Regular Window</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Borderless Window</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>416</width>
        <height>383</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Advanced</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../qutim.qrc" >
        <normaloff>:/icons/crystal_project/advanced.png</normaloff>:/icons/crystal_project/advanced.png</iconset>
      </attribute>
      <attribute name="toolTip" >
       <string>Advanced settings</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="sysTrayStatusBox" >
         <property name="title" >
          <string>System tray status</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="noSysTrayButton" >
            <property name="text" >
             <string>No tray status</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QRadioButton" name="statusFromButton" >
              <property name="text" >
               <string>Status from:</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="trayAccountBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="inSysTrayBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Show in system tray menu</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Style:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QComboBox" name="styleComboBox" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <item>
          <property name="text" >
           <string>Default</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Translation:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QComboBox" name="translationComboBox" >
         <item>
          <property name="text" >
           <string>English</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Lithuanian</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Russian</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Ukrainian</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0" >
        <widget class="QComboBox" name="sIconsComboBox" >
         <item>
          <property name="text" >
           <string>Default</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0" >
        <widget class="QLabel" name="sIconsLabel" >
         <property name="text" >
          <string>Status Icons:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>216</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../qutim.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>showSettingsBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>accountBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>208</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel" >
     <x>296</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>statusFromButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>trayAccountBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>110</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel" >
     <x>110</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>statusFromButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>inSysTrayBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>110</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel" >
     <x>94</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ahideBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>secondsBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>79</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel" >
     <x>136</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showGroupsBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>hideGroupsBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel" >
     <x>228</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
